blog-content/lw5-d3-metrics.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Day 3: Metrics & APM"
-createdAt: 2024-02-02T12:00:00Z
+createdAt: 2024-04-29T12:00:00Z
```

Comment:
can we keep everything at/after day 3 as older? that way they're essentially hidden.